content/chat/moderation.textile

```diff
@@ -1,7 +1,38 @@
 ---
 title: Moderation
-meta_description: "Limit chat participant’s ability to post messages within a given timeframe within specific conversations, or entire conversation namespaces."
+meta_description: "Limit chat participant's ability to post messages within a given timeframe within specific conversations, or entire conversation namespaces."
 product: chat
 languages:
   - javascript
 ---
+
+As with any chat applications, there needs to be some functionality implemented to allow for moderation to ensure the chat participants don't breach the terms of use or carry out actions such as spamming conversations.
+
+h2(#slow-mode). Slow mode
+
+Large-scale live-streaming conversations may result in high volumes of participants having difficulties keeping up with the flow of the conversations. Every chat platform also works hard to fight against the risk of those with malicious intent on spamming their channels. Slow mode enables you to prevent users from sending an excessive number of messages in a short period of time.
+
+Chat participants can be restricted either on individual conversations, or as a set of channels defined by a common namespace. For example if you set the value of @ably.limits.publish.perAttachment.maxRate.chat:*@ to one of @1@, then chat participants would only be able publish one message per second in any conversation defined under the @chat@ namespace. Examples of other possibilities are shown in the table below:
+
+|_=. maxRate Value |_=. Message rate |
+| 0.1 | 1 every 10 seconds |
+| 1 | 1 per second |
+| 5 | 5 per second |
+| 10 | 10 per second |
+
+h3(#set-limits). Set limits
+
+Limits are set in the auth token which are generated from a trusted device that has access to the API key. The limits are defined within the claims in your JWT authentication payload using the key @ably.limits.publish.perAttachment.maxRate.<resource-name>@.
+
+For example, to limit chat participants to only be able to publish 10 messages per second in the @chat1@ room, but 1 message every 10 seconds in any conversation under the @chat@ namespace, your claims object would appear similar to what's shown in the example below:
```

Comment:
It's per-attachment, so doesn't affect REST API or multiple attachments by a single user to the same channel

Reply:
https://github.com/ably/docs/pull/2090/commits/f2e0d2ea9b18a8b2d640a9d65d54b1a783ee5c4a